### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by chlane**
Verbiage for #1084

[Menu preview](https://raw.githack.com/chlane/aria/main/index.html#menu)

[Menubar preview](https://raw.githack.com/chlane/aria/main/index.html#menubar)

***
<a href="https://pr-preview.s3.amazonaws.com/chlane/aria/pull/1743.html" title="Last updated on Jul 19, 2022, 3:35 PM UTC (c9baac7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1743/7999ec7...chlane:c9baac7.html" title="Last updated on Jul 19, 2022, 3:35 PM UTC (c9baac7)">Diff</a>

### CODE REVIEW

**Comment by jnurthen** [comment]
@chlane can you please add your github ID in your w3c profile page https://www.w3.org/users/135607

**Comment by chlane** [comment]
Hi @jnurthen my Github account chlane is connected at https://www.w3.org/users/135607/connectedaccounts.

File: index.html [hunk 1]
```diff
@@ -5300,8 +5300,9 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 		<div class="role" id="menu">
 			<rdef>menu</rdef>
 			<div class="role-description">
-				<p>A type of <a>widget</a> that offers a list of choices to the user.</p>
-				<p>A menu is often a list of common actions or functions that the user can invoke. The <code>menu</code> <a>role</a> is appropriate when a list of menu items is presented in a manner similar to a menu on a desktop application.</p>
+				<p>A type of <a>widget</a> that offers a list of choices to the user. A menu is <em class="rfc2119">usually</em> presented vertically and <em class="rfc2119">usually</em> does not remain visible.
```
Comment (jnurthen) [inline index.html:5303]: usually is not one of the rfc2119 keywords. It should not have the class="rfc2119".

File: index.html [hunk 2]
```diff
@@ -5398,7 +5399,7 @@ <h4>Plain HTML or Polyfill DOM Result of the MathML Quadratic Formula</h4>
 		<div class="role" id="menubar">
 			<rdef>menubar</rdef>
 			<div class="role-description">
-				<p>A presentation of <rref>menu</rref> that usually remains visible and is usually presented horizontally.</p>
+				<p>A presentation of <rref>menu</rref> that <em class="rfc2119">usually</em> remains visible. Operating systems and applications may allow users to temporarily hide menu bars. Additionally, a menubar is <em class="rfc2119">usually</em> presented horizontally.</p>
```
Comment (jnurthen) [inline index.html:5402]: please remove the class on usually

**Review by jnurthen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**